src/host/getset.cpp

```diff
@@ -374,6 +374,16 @@ void ApiRoutines::GetNumberOfConsoleMouseButtonsImpl(ULONG& buttons) noexcept
             // ECHO on with LINE off is invalid.
             RETURN_HR_IF(E_INVALIDARG, WI_IsFlagSet(mode, ENABLE_ECHO_INPUT) && WI_IsFlagClear(mode, ENABLE_LINE_INPUT));
         }
+        if (WI_IsFlagSet(mode, ENABLE_MOUSE_INPUT))

```

Comment:
should we do something like the `if (gci.GetInsertMode() != PreviousInsertMode)` lines above, where we only emit these sequences when they _changed_ `ENABLE_MOUSE_INPUT`? Might be less chatty

Reply:
That makes sense!